### PR TITLE
Added movie show with links

### PR DESCRIPTION
**Pull request by markiecyt**

### CODE REVIEW

File: app/assets/stylesheets/components/_cards.scss [hunk 1]
```diff
@@ -47,11 +47,10 @@
   }
   h2 {
     font-weight: bold;
+    font-size: 24px;
     text-shadow: 1px 1px 3px rgba(0,0,0,0.2);
     color: lightgray;
-    span {
-      font-size: 14px;
-    }
+
```
Comment (JupitersLB) [inline app/assets/stylesheets/components/_cards.scss]: Is this empty line necessary?

File: app/views/movies/show.html.erb [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+          <span>(<%= @movie.rating %>/10)</span>
+        </h2>
+        <p><%= @movie.overview %></p>
+        <div class="cards">
+          <% @movie.lists.each do |list| %>
+            <%= render partial: "lists/card", locals: {list: list} %>
+          <% end %>
+        </div>
+      </div>
+
```
Comment (JupitersLB) [inline app/views/movies/show.html.erb]: Same here

File: config/routes.rb [hunk 3]
```diff
@@ -1,7 +1,9 @@
 Rails.application.routes.draw do
+  get 'movies/show'
```
Comment (JupitersLB) [inline config/routes.rb]: should remove this if you are also using resources

File: app/views/movies/show.html.erb [hunk 4]
```diff
@@ -0,0 +1,20 @@
+<div class="container pt-5">
+  <div class="movie-page d-flex mt-5 py-5">
+    <img src="<%= @movie.poster_url %>" alt="Movie image" class="movie-image d-flex">
```
Comment (JupitersLB) [inline app/views/movies/show.html.erb:3]: Need to make this responsive

**Review by JupitersLB — CHANGES_REQUESTED** [review]

File: app/views/movies/show.html.erb (hunk 4 above)
Comment (JupitersLB) [reply app/views/movies/show.html.erb:3]: Could make responsive changes in a new branch

Comment (markiecyt) [reply app/views/movies/show.html.erb:3]: Yes, okay, will do! Thanks!

**Comment by markiecyt** [comment]
@JupitersLB Fixed, please check.

**Review by JupitersLB — APPROVED** [review]
LGTM

**Pull request merged**